library/psa_crypto.c

```diff
@@ -2411,27 +2411,27 @@ static psa_status_t psa_mac_init( psa_mac_operation_t *operation,
 {
     psa_status_t status = PSA_ERROR_NOT_SUPPORTED;
 
-    operation->alg = PSA_ALG_FULL_LENGTH_MAC( alg );
-    operation->key_set = 0;
-    operation->iv_set = 0;
-    operation->iv_required = 0;
-    operation->has_input = 0;
-    operation->is_sign = 0;
+    operation->ctx.mbedtls_ctx.alg = PSA_ALG_FULL_LENGTH_MAC( alg );
```

Comment:
I'll keep it in mind for future PRs

Reply:
You have to rebase anyway thus please do it in this PR. Otherwise, I will have to check the code move line by line to be sure that nothing like below (line 2560) happened.

Reply:
There's quite a difference in workload between a rebase to resolve merge conflicts and a fairly extensive rewrite (what you're asking for). It sounds like you care deeply, so I guess I'll do it, but I don't really see the point.

Reply:
The point is to facilitate review. Generally speaking, a commit should either do a single piece of refactoring or do a behavior change. Renaming identifiers and moving code are two different kinds of refactoring, so they shouldn't be done in the same commit.

Reply:
Done.